lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2492,6 +2492,14 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
   case OpSignBitSet:
     return mapValue(BV,
                     transRelational(static_cast<SPIRVInstruction *>(BV), BB));
+  case OpIAddCarry: {
+    IRBuilder Builder(BB);
+    auto *BC = static_cast<SPIRVBinary *>(BV);
+    return mapValue(BV, Builder.CreateBinaryIntrinsic(
```

Comment:
Please look at comment in SPIRVWriter.cpp. In the reverse translation, the second LLVM type will always be i1.

Thanks

Reply:
Done